### ISSUE TITLE
`airflow pools set` fails with argparse error

### ISSUE THREAD

**Issue by robmoore** [apache/airflow#14940]
**Apache Airflow version**:
1.10.14 and 1.10.15

**Environment**:

- **Cloud provider or hardware configuration**: Docker 
- **OS** (e.g. from /etc/os-release): apache/airflow:1.10.15-python3.8
- **Install tools**: poetry

**What happened**:

Based on a warning from Airflow, I decided to change from 

`airflow pool -s mypoolname 2 mypooldesc`

to 

`airflow pools set mypoolname 2 mypooldesc`

The first command was successful while the second resulted in the following error:

```
airflow_1   | Traceback (most recent call last):
airflow_1   |   File "/home/airflow/.local/bin/airflow", line 37, in <module>
airflow_1   |     args.func(args)
airflow_1   |   File "/home/airflow/.local/lib/python3.8/site-packages/airflow/bin/cli.py", line 302, in pool_set
airflow_1   |     _pool_wrapper(args, set=(args.name, args.slots, args.description))
airflow_1   | AttributeError: 'Namespace' object has no attribute 'name'
```

**What you expected to happen**:

I expected it to have the same effect as running the to-be-deprecated version.

**How to reproduce it**:

1. Start up Airflow
2. Attempt to create a pool using `airflow pools set <name> <number of slots> <description>`

**Anything else we need to know**:

The error occurs consistently. Switching back to the to-be-deprecated version works every time.

**Comment by robmoore** [comment]
@kaxil Tagging you per your request in [Slack group discussion](https://apache-airflow.slack.com/archives/CCQ7EGB1P/p1616445607186200).

**Comment by XD-DENG** [comment]
Maybe another pool-related CLI, `pool_delete`, is also wrong? `delete=pool.name` should be `delete=args.pool`
https://github.com/apache/airflow/blob/5786dcdc392f7a2649f398353a0beebef01c428e/airflow/bin/cli.py#L301-L306

**Comment by lordozb** [comment]
@kaxil 
Is it okay if you assign this to me?

**Comment by XD-DENG** [comment]
Hi @lordozb  here you go.

**Comment by lordozb** [comment]
@XD-DENG 
Here is the pull request https://github.com/apache/airflow/pull/15056

**Comment by XD-DENG** [comment]
Fixed by #15056

**Comment by Jorricks** [comment]
Based on this issue. 
Are there any plans on releasing a v1.10.16?
